scenario-compiler/src/main/scala/pl/touk/nussknacker/engine/definition/clazz/ClassDefinitionSet.scala

```diff
@@ -10,13 +10,11 @@ object ClassDefinitionSet {
 
   // Below are for tests purpose mostly
   def forClasses(classes: Class[_]*): ClassDefinitionSet = ClassDefinitionSet(
-    ClassDefinitionDiscovery.discoverClasses(classes ++ ExpressionConfig.defaultAdditionalClasses)(
-      ClassExtractionSettings.Default
-    )
+    ClassDefinitionDiscovery.Default.discoverClasses(classes ++ ExpressionConfig.defaultAdditionalClasses)
```

Comment:
WDYT about moving this test-only classes to some util in test utils module? It will be easier to detect not intended access to `ClassExtractionSettings.Default`

Reply:
sure

Reply:
Unfortunatley is not possible because in a such situation test module would have dependency to ScenarioCompiler and then tests in ScenarioCompile could not have dependency to this test module due to cyclic dependency.